integrations/cdi/jta-cdi/pom.xml

```diff
@@ -88,6 +88,10 @@
             <artifactId>helidon-common-features-api</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>io.helidon.common</groupId>
+            <artifactId>helidon-common</artifactId>
+        </dependency>
```

Comment:
It would be great if we could avoid requiring helidon common here, since everything else about the integration is not Helidon-specific (not sure where the features dependency came in)

Reply:
We have already agreed with @romain-grecourt that we don't want separated spi module for our CraC abstraction